packages/mds-geography/api.ts

```diff
@@ -0,0 +1,185 @@
+import express from 'express'
+import db from '@mds-core/mds-db'
+
+import { pathsFor, ServerError, NotFoundError } from '@mds-core/mds-utils'
+import { geographyValidationDetails } from '@mds-core/mds-schema-validators'
+import log from '@mds-core/mds-logger'
+
+import { checkAccess } from '@mds-core/mds-api-server'
+
+function api(app: express.Express): express.Express {
+  app.get(
+    pathsFor('/geographies/meta/'),
+    checkAccess(scopes => scopes.includes('policies:read')),
+    async (req, res) => {
+      const get_read_only = req.query === 'true'
```

Comment:
I don't think this behaves properly. `req.query` returns an object mapping of `{ query: value }`, so for example, a URL of `somewebsite.com/thing?foo=iamfoo&bar=iambar` would result in `req.query = { foo: 'iamfoo', bar: 'iambar' }`. FWIW, The express typedefs here are a bit loose, as the type of `req.query` is `any`. 

What this snippet is doing, is checking to see if `req.query === 'true'` which will _never_ be the case. I think what you want is this:
`const get_read_only = req.query.get_read_only === 'true'`